### PR TITLE
[Tizen.platform] make file names available to be changed

### PR DESCRIPTION
**Pull request by helloahn**
Rather than the fixed paths, set file path through arguments

Signed-off-by: HyoungJoo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by myungjoo** [comment]
@leemgs @helloahn

In the next release, please locate V's model file at
```/usr/lib/nnstreamer/vivante_models/${modelname}/``` for both .nb and .so.

**Comment by leemgs** [comment]
We have been putting the *.so in the /usr/share/vivante/ folder.

**Comment by myungjoo** [comment]
> nte/ folder.



> We have been putting the *.so in the /usr/share/vivante/ folder.

That's fine, too.
(```/usr/share/vivante/${modelname}/*.nb, *.so```)

File: Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md [hunk 1]
```diff
@@ -8,30 +8,37 @@ you can check the rpm files at your ${GBS-ROOT}
 
 After gbs build and install at your target, you can execute it with below command.
 ```bash
-$ vivante-inceptionv3-pipeline ${PATH_TO_JPEG}
-# e.g. vivante-inceptionv3-pipeline /usr/share/vivante-inceptionv3-pipeline/sample_pizza_299x299.jpg
+$ vivante-inceptionv3-pipeline ${PATH_TO_NB_FILE} ${PATH_TO_SO_FILE} ${PATH_TO_JPEG}
+# e.g. $ vivante-inceptionv3-pipeline \
+> /usr/share/dann/inception-v3.nb \
```
Comment (leemgs) [inline Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md]: @helloahn , The example paths need to be updated consistently.  For example, 
* DA uses the **/usr/share/dann/** folder.
* SR uses the **/usr/share/vivante/** folder.

FYI, Now, image (*.jpg) and .nb is located in the /usr/share/dann/ (DA) folder. And the .so library for .nb is located in  the /usr/share/vivante/ (SR) folder.

File: Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md [hunk 2]
```diff
@@ -8,30 +8,37 @@ you can check the rpm files at your ${GBS-ROOT}
...
+> /usr/lib/nnstreamer/filters/libinceptionv3.so \
+> /usr/share/vivante-inceptionv3-pipeline/sample_pizza_299x299.jpg
 ```
 
 The result should be similar with below
 ```bash
-root@localhost:~# vivante-inceptionv3-pipeline /usr/share/vivante-inceptionv3-pipeline/sample_pizza_299x299.jpg
+root@localhost:~# vivante-inceptionv3-pipeline \
```
Comment (leemgs) [inline Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md]: Impeccable.

File: Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/src/main.c [hunk 3]
```diff
@@ -13,16 +13,19 @@
...
 #define MODEL_HEIGHT  299
 #define CH  3
 #define LABEL_SIZE  5
 
 gchar * pipeline;
-gchar * res_path;
 gchar * img_path;
-gchar * model1_path;
-gchar * model2_path;
+gchar * model_nb;
```
Comment (leemgs) [inline Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/src/main.c]: Looks better for readability. :)

File: Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/src/main.c [hunk 4]
```diff
@@ -101,21 +103,21 @@ get_label_from_tensor_sink (const ml_tensors_data_h data,
  */
 int main(int argc, char *argv[]){
   int status;
-  img_path = g_strdup_printf("%s", argv[1]);
-  model1_path = g_strdup_printf("/usr/share/dann/inception-v3.nb");
-  model2_path = g_strdup_printf("/usr/lib/nnstreamer/filters/libinceptionv3.so");
+  model_nb = g_strdup_printf("%s", argv[MODEL_NB]);
```
Comment (leemgs) [inline Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/src/main.c]: Neat.

File: Tizen.platform/Vivante_single_NonGUI_inceptionV3/README.md [hunk 5]
```diff
@@ -8,8 +8,11 @@ you can check the rpm files at your ${GBS-ROOT}
 
 After gbs build and install at your target, you can execute it with below command.
 ```bash
-$ vivante-inceptionv3-single ${PATH_TO_INPUT_FILE}
-# e.g. vivante-inceptionv3-single /usr/share/vivante-inceptionv3-single/sample_pizza_299x299
+$ vivante-inceptionv3-single ${PATH_TO_NB_FILE} ${PATH_TO_SO_FILE} ${PATH_TO_INPUT_FILE}
```
Comment (leemgs) [inline Tizen.platform/Vivante_single_NonGUI_inceptionV3/README.md]: Please refer to the comment above.

File: Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md (hunk 1 above)
Comment (helloahn) [reply Tizen.platform/Vivante_pipeline_NonGUI_inceptionV3/README.md]: I agree. I'll unify the path of the sample resources.

**Pull request merged**